src/main/java/org/junitpioneer/jupiter/EnvironmentVariableExtension.java

```diff
@@ -92,8 +92,9 @@ private void storeOriginalEnvironmentVariables(ExtensionContext context, Collect
 
 	private void reportWarning(ExtensionContext context) {
 		boolean wasReported = REPORTED_WARNING.getAndSet(true);
-		if (!wasReported)
+		if (!wasReported) {
 			context.publishReportEntry(WARNING_KEY, WARNING_VALUE);
+		}
```

Comment:
I don't like having these superfluous parenthesis in our code. They add little except almost empty lines.

Reply:
but i do not think they are sonar related - sonar is not complaining about those in the gui

Reply:
Maybe I just saw this while doing something else in the code and edited it then. For me (personally) they improve readability of the code. And I like them because I had to debug an massive amount of bugs where at the end it comes down that someone thought "na I don't need parantheses.." and broke the code.

But as @aepfli mentions there is no rule which enforces the one or the other, we can go with both as the author wants to use it.

Reply:
I just checked it. It was added by my `save action` plugin. I have only a very few of its action enabled, but as written that I really like having paranthesis for blocks.

But as wished I disabled it to revert this one. So now at this if the paranthesis are gone again, but in a similar block a few lines below they are still there so we have an inconsistency there now (which I never had touched, so they are from the original author).

So you see were this ends? We should either define if we want to force using paranthesis or not for blocks and force them by a rule or we don't force the one or the other and let the authors use their personal like.